.gitignore

```diff
@@ -1,2 +1,4 @@
 out
 node_modules
+.vscode-test
+package-lock.json
```

Comment:
According to the npm documentation, package-lock should be commited (https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md)

Reply:
Package-lock is quite annoying, and there is a debate whether it should or not belong to repos. A few big repos don't use it, for example, https://github.com/browserify/browserify

But agreed, we should stick to NPM recommendations, and I'll restore the package-lock for this commit.